docs/jsclientlib/index.rst

```diff
@@ -14,7 +14,7 @@ If you are using it from a web page hosted on OctoPrint as a Jinja2 template, yo
 methods to embed it instead of manually entering the URL, in order to have OctoPrint take care of setting the
 correct URL prefix:
 
-.. code-block:: html
+.. code-block:: html+jinja
```

Comment:
😱 I had no idea this was possible! Thanks for showing me!

Reply:
It took me some searching through the huge Pygments lexers page, but yes, those things exist :)
http://pygments.org/docs/lexers/